frontend/src/components/App/Settings/SettingsCluster.tsx

```diff
@@ -222,27 +266,56 @@ export default function SettingsCluster() {
     "translation|Cluster name must contain only lowercase alphanumeric characters or '-', and must start and end with an alphanumeric character."
   );
 
+  // If we don't have yet a cluster name from the URL, we are still loading.
+  if (!clusterFromURLRef.current) {
+    return <Loader title="Loading" />;
+  }
+
+  if (clusters.length === 0) {
+    return (
+      <>
+        <SectionBox title={t('translation|Cluster Settings')} backLink />
+        <Empty color="error">{t('translation|There seem to be no clusters configured…')}</Empty>
+      </>
+    );
+  }
+
+  if (!cluster) {
+    return (
+      <>
+        <SectionBox title={t('translation|Cluster Settings')} backLink>
+          <ClusterSelector clusters={clusters} />
```

Comment:
What do you think of placing it as a sub heading?
```ts
        <SectionBox title={t('translation|Cluster Settings')} backLink>
          <Typography color="error" component="h3" variant="h5">
            {t(
              'translation|Cluster {{ clusterName }} does not exist. Please select a valid cluster.',
              {
                clusterName: clusterFromURLRef.current,
              }
            )}
          </Typography>
          <ClusterSelector clusters={clusters} />
        </SectionBox>

```

![image](https://github.com/user-attachments/assets/d5557152-4c34-44ed-ba5b-3ad1131e84b0)

Reply:
Note: for the error color contrast issue I made a PR https://github.com/headlamp-k8s/headlamp/pull/2284 

So the red is a bit lighter and more readable.

Reply:
I think placing it as a sub-heading might be appealing, especially if we end the message with a colon like this:
#### Please select a valid cluster:

Played around with the colors as well, I think the alpha on `error.light` is a little too low to read on the dark background comfortably, so I think the main error color will suffice for now. Maybe I can revisit that? looking through the lighthouse results as well